libraries/standard.js

```diff
@@ -53,6 +53,20 @@ const null_check = (dst, data, data_field, skip_errors, errors) => {
   return false  
 }
 
+// takes the whole details object and removes any null values
+// use case: item.details = remove_if_null(details);
+const remove_if_null = function(details_obj){
+  // go through the details object and remove any null values
+  Object.keys(details_obj).forEach(key => {
+      if (details_obj[key] === null) {
```

Comment:
this should check for undefined or empty string as well, see https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in